week_2/pet.rb

```diff
@@ -90,26 +95,40 @@ def food_cost(days)
   # snake = Pet.new(name: 'python', animal_type_id: 4, food_consumed_per_day: 0.3)
   # Pet.cost_to_feed([cat, dog, fish, snake], 6) will return 6180.0
   def self.cost_to_feed(pets, days)
-    raise NotImplementedError # TODO
+    c=0
+   
+  pets.each do |an|
+    c=c+(FOOD_COST_PER_KG[an.animal_type_id]*days*an.food_consumed_per_day)
+  end
```

Comment:
[reduce](https://medium.com/@james.a.hughes/using-the-reduce-method-in-ruby-907f3c18ae1f) method can be used here.